schema/partner_show.js

```diff
@@ -9,7 +9,7 @@ import total from '../lib/loaders/total';
 import { exhibitionPeriod, exhibitionStatus } from '../lib/date';
 import cached from './fields/cached';
 import date from './fields/date';
-import markdown from './fields/markdown';
+import { markdown } from './fields/markdown';
```

Comment:
You'll have to do this in other places that require markdown now that there is no default export in that file. The other option is to export the original function as default (like it was before) and do something like:

```
import markdown, { formatMarkdownValue } from '../fields/markdown';
```